modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/UpdatedEventSyncCommand.java

```diff
@@ -87,27 +88,34 @@ private void doSyncMedia( final ContentEventSyncCommandParams params, final Upda
 
     private void doSyncThumbnail( final ContentEventSyncCommandParams params, final UpdateContentParams updateParams )
     {
-        if ( params.getSourceContent().hasThumbnail() &&
-            !params.getSourceContent().getThumbnail().equals( params.getTargetContent().getThumbnail() ) )
+        if ( !Objects.equals( params.getSourceContent().getThumbnail(), params.getTargetContent().getThumbnail() ) )
         {
             final Thumbnail sourceThumbnail = params.getSourceContent().getThumbnail();
+            final Thumbnail targetThumbnail = params.getTargetContent().getThumbnail();
```

Comment:
Please to declare the `targetThumbnail` variable in the `else` block, if `sourceThumbnail` is not null then that variable not needed.